lake/sequence_generator.py

```diff
@@ -0,0 +1,38 @@
+"""SequenceGenerator class."""
+import numpy as np
+import string
+
+class SequenceGenerator:
+  """XXXXXX"""
+  letters = np.array(list(string.ascii_uppercase))
+
+  def __init__(self, characters, seq_length):
```

Comment:
It seems like `characters` must be even.. which maybe does make sense.

Can you add some validation check in the init method, that ensures that `characters` is even, otherwise throw an error with a useful error message?

Reply:
If user provides an odd number, we add 1 and throw a message notifying the change.